hackathonbaobab2021/core/instance.py

```diff
@@ -15,19 +15,29 @@
 )
 
 C_CAT = SuperDict(
-    CA1=["slots"],
+    CA1=["slots", "teams"],
+    CA2=["slots", "teams1", "teams2"],
     CA3=["teams1", "teams2"],
+    CA4=["teams1", "teams2", "slots"],
     GA1=["meetings", "slots"],
+    BR1=["teams", "slots"],
     BR2=["slots", "teams"],
     SE1=["teams"],
+    FA2=["teams", "slots"],
 )
 C_TUPLES = SuperDict(GA1=SuperDict(meetings=True))
 
 _ID = "_id"
 _CAT = "_cat"
 
+INT_PROPS = ["min", "max", "intp", "penalty"]
```

Comment:
Should we move these constants as the ones in `experiment.py` to a `const.py` file in the core folder? Does it make sense?

Reply:
Yes, it makes sense. I will do that.